lib/plugin.js

```diff
@@ -1,15 +1,13 @@
-module.exports = function pluginMethod (plugin) {
-  var self = this
-
+module.exports = function pluginMethod (hoodie, options, plugin) {
```

Comment:
`pluginMethod (hoodie, options, plugin)` changes to `pluginMethod (options, plugin)`